web/src/views/form.tsx

```diff
@@ -189,15 +189,21 @@ const MyForm: React.SFC = () => {
 
     useEffect(() => {
         async function wrap() {
-            const result = await axi.get('ministry', {
-                headers: {
-                    Accept: 'application/json',
-                },
-            });
-
-            if (result.data) {
-                console.log('Fetched ministry sponsors!!!');
-                setMinistrySponsor(result.data);
+            try {
+                const result = await axi.get('ministry', {
```

Comment:
you'll have to test this but I'm pretty sure `axios` does not through on `4xx` http status codes, so you may need to check that as well

Reply:
The error handling is added here to make sure unit tests can run successfully and independently on the UI side. I am not sure what to do. We could discuss this if you like.